lib/miq_automation_engine/models/miq_ae_datastore/xml_import.rb

```diff
@@ -155,8 +155,8 @@ def self.load_xml(f, domain_name = nil)
     end
 
     def self.create_domain(domain)
-      ns = MiqAeNamespace.find_by_fqname(domain)
-      MiqAeNamespace.create!(:name => domain, :enabled => true, :priority => 100) unless ns
+      ns = MiqAeDomain.where(:name => domain).first
+      MiqAeDomain.create!(:name => domain, :enabled => true, :priority => 100, :tenant => Tenant.seed) unless ns
```

Comment:
Can this whole method just become:
`MiqAeDomain.find_or_create_by!(:name => domain, :enabled => true, :priority => 100, :tenant => Tenant.seed)`

Otherwise, I would suggest using `find_by` here instead of `where().first`